<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="device">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>361</width>
      <height>451</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="up_button">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>300</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="left_button">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>340</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="right_button">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>340</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="down_button">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>380</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ok_button">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>340</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ok</string>
     </property>
    </widget>
    <widget class="QFrame" name="screen_frame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>321</width>
       <height>211</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QStackedWidget" name="screen">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>301</width>
        <height>191</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="menu">
       <widget class="QListWidget" name="menu_screen">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>221</width>
          <height>91</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Start a session</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Settings</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Logs/History</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="menu_title">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>54</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Menu</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="settings">
       <widget class="QListWidget" name="settings_screen">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>221</width>
          <height>91</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Challenge level</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Breath Pacer Settings</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="setttings_title">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>54</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="sessions_page">
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>261</width>
          <height>141</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="challenge_screen">
       <widget class="QListWidget" name="challenge_list">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>221</width>
          <height>81</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="challenge_title">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>0</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Challenge Level</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page">
       <widget class="QTextBrowser" name="textBrowser_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>291</width>
          <height>191</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="achievement_box">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>30</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0.0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="coherence_box">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0.0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="length_box">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0.0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Coherence</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>10</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Achievement</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>54</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Length</string>
        </property>
       </widget>
       <widget class="QCustomPlot" name="Plot" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>261</width>
          <height>101</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QTextBrowser" name="textBrowser_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>301</width>
          <height>191</height>
         </rect>
        </property>
       </widget>
       <widget class="QCustomPlot" name="Plot_2" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>261</width>
          <height>111</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>29</x>
          <y>0</y>
          <width>241</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" alignment="Qt::AlignLeft">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" alignment="Qt::AlignLeft">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="back_button">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>280</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="on_off_button">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>410</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="menu_button">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>280</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextBrowser" name="light_box">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="coherence_table">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>540</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>1.0</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>1.6</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>2.4</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>2.0</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>3.0</string>
     </property>
    </item>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_15">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>580</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
